tests/unittests/distros/test_create_users.py

```diff
@@ -1,342 +1,294 @@
 # This file is part of cloud-init. See LICENSE file for license information.
 
-import re
-
-from cloudinit import distros, log, ssh_util
-from tests.unittests.helpers import CiTestCase, mock
-from tests.unittests.util import abstract_to_concrete
+from typing import List
 
+import pytest
 
-@mock.patch("cloudinit.distros.util.system_is_snappy", return_value=False)
-@mock.patch("cloudinit.distros.subp.subp")
-class TestCreateUser(CiTestCase):
```

Comment:
Thank you for for refactoring away from CiTestCase. Yay!! An angel just got its wings.